crates/uv-cli/src/lib.rs

```diff
@@ -610,14 +619,30 @@ pub struct VersionArgs {
     pub python: Option<Maybe<String>>,
 }
 
-#[derive(Debug, Copy, Clone, PartialEq, clap::ValueEnum)]
+// Note that the ordering of the variants is significant, as when given a list of operations
+// to perform, we sort them and apply them in order, so users don't have to think too hard about it.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, clap::ValueEnum)]
 pub enum VersionBump {
     /// Increase the major version (1.2.3 => 2.0.0)
     Major,
     /// Increase the minor version (1.2.3 => 1.3.0)
     Minor,
     /// Increase the patch version (1.2.3 => 1.2.4)
     Patch,
+    /// Make the version stable (1.2.3b4.post5.dev6 => 1.2.3)
+    ///
+    /// This intentionally clears `.postN` and preserves `+local`
```

Comment:
I might mention here that this could decrease the version number, and therefore, it's common to use this in conjunction with another `--bump` flag like `--bump patch`.